core/ledger/kvledger/txmgmt/rwsetutil/rwset_proto_util_test.go

```diff
@@ -63,6 +158,51 @@ func TestTxRWSetMarshalUnmarshal(t *testing.T) {
 	testutil.AssertNoError(t, txRwSet1.FromProtoBytes(protoBytes), "")
 	t.Logf("txRwSet=%s, txRwSet1=%s", spew.Sdump(txRwSet), spew.Sdump(txRwSet1))
 	testutil.AssertEquals(t, txRwSet1, txRwSet)
```

Comment:
swap the 2nd and 3rd arguments. I think that the 3rd argument to this function is 'expected value'